eng/Version.Details.xml

```diff
@@ -1,26 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Razor.Symbols.Transport" Version="7.0.0-preview.5.22528.1">
-      <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>0d427c7a238c5b7b114802459db7fc56bea6357d</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.CodeAnalysis.Razor.Tooling.Internal" Version="7.0.0-preview.5.22528.1">
-      <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>0d427c7a238c5b7b114802459db7fc56bea6357d</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Tooling.Internal" Version="7.0.0-preview.5.22528.1">
-      <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>0d427c7a238c5b7b114802459db7fc56bea6357d</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.Internal.Transport" Version="7.0.0-preview.5.22528.1">
-      <Uri>https://github.com/dotnet/razor-compiler</Uri>
-      <Sha>0d427c7a238c5b7b114802459db7fc56bea6357d</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.Internal.Transport" Version="7.0.0-preview.5.22528.1">
-      <Uri>https://github.com/dotnet/razor-compiler</Uri>
```

Comment:
I wonder how many other repos are still incorrectly taking a dependency on this repo. Going to try and dig into this.